src/App.test.tsx

```diff
@@ -2,8 +2,7 @@ import React from "react";
 import { render, screen } from "@testing-library/react";
 import App from "./App";
 
-test("renders learn react link", () => {
-  render(<App />);
-  const linkElement = screen.getByText(/What to watch/i);
-  expect(linkElement).toBeInTheDocument();
+test("should render App", () => {
+  render(<App/>);
+  screen.debug();
```

Comment:
test if some element is rendered

Reply:
it is a good idea to add a new test, but you need to update this one

Reply:
updated